buildSrc/src/main/groovy/com/uber/okbuck/core/model/AndroidTarget.groovy

```diff
@@ -257,7 +261,8 @@ abstract class AndroidTarget extends JavaLibTarget {
             XmlSlurper slurper = new XmlSlurper()
             GPathResult manifestXml = slurper.parse(project.file(mergedManifest))
 
-            manifestXml.@package = applicationId + applicationIdSuffix
+            packageName = manifestXml.@package
```

Comment:
I dont think this is correct.

See http://tools.android.com/tech-docs/new-build-system/applicationid-vs-packagename